src/include/sof/audio/buffer.h

```diff
@@ -249,8 +249,7 @@ void buffer_detach(struct comp_buffer *buffer, struct list_item *head, int dir);
 
 static inline struct comp_dev *buffer_get_comp(struct comp_buffer *buffer, int dir)
 {
-	struct comp_dev *comp = dir == PPL_DIR_DOWNSTREAM ? buffer->sink :
-		buffer->source;
+	struct comp_dev *comp = dir == PPL_DIR_DOWNSTREAM ? buffer->sink : buffer->source;
```

Comment:
This one was already changed in part1: https://github.com/thesofproject/sof/pull/8206 , didn't notice.

Reply:
No problem to keep if commit message is updated for cosmetic change only.